src/themes/capybara/components/category/FiltersSidebar.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <SfSidebar
+    :visible="visible"
+    @close="onClose"
+    class="filters"
+  >
+    <h3 class="filters__title">
+      Color
+    </h3>
+    <SfFilter
```

Comment:
When user adds additional fields to `config.products.defaultFilters` they should be also rendered using kind of general filter (`GenericSelector` in the current theme) -> https://github.com/DivanteLtd/vue-storefront/blob/e486c368b51fbbc20f29e3cdda68e4c24ef5fe98/src/themes/default/components/core/blocks/Category/Sidebar.vue#L63

Reply:
imho this should be part of vuex then to filter it @patzick . The question is do we really need this or want to leave it up to developers for better flexibility